src/components/HeroSection/Hero.jsx

```diff
@@ -0,0 +1,25 @@
+
+function Hero() {
+  return (
+    <div className='backdrop-blur-md bg-white/30 h-[100vh] cont text-center text-background'>
```

Comment:
adding h-screen instead of h-[100vh] doesn't work?

Reply:
It works but its the same